src/sc2api/CMakeLists.txt

```diff
@@ -9,26 +6,33 @@ file(WRITE "${PROJECT_SOURCE_DIR}/include/sc2api/sc2_typeenums.h"
 "
 )
 
-list(APPEND sources_sc2
-    "${PROJECT_SOURCE_DIR}/include/sc2api/sc2_typeenums.h"
-    "${PROJECT_SOURCE_DIR}/include/sc2api/typeids/sc2_${SC2_VERSION}_typeenums.h"
-    "${PROJECT_SOURCE_DIR}/src/sc2api/typeids/sc2_${SC2_VERSION}_typeenums.cpp"
-)
-
-add_library(sc2api ${sources_sc2})
-
-target_include_directories(sc2api
-    PUBLIC
-        "${PROJECT_SOURCE_DIR}/include"
-        "${PROJECT_SOURCE_DIR}/contrib/civetweb/include")
-
-target_link_libraries(sc2api
-    PUBLIC
-        sc2protocol
-        civetweb-c-library)
-
-set_target_properties(sc2api PROPERTIES DEBUG_POSTFIX "d")
+set(sc2api_sources
+    sc2_action.cc
+    sc2_agent.cc
+    sc2_args.cc
+    sc2_client.cc
+    sc2_common.cc
+    sc2_connection.cc
+    sc2_coordinator.cc
+    sc2_data.cc
+    sc2_game_settings.cc
+    sc2_map_info.cpp
+    sc2_proto_interface.cc
+    sc2_proto_to_pods.cc
+    sc2_replay_observer.cc
+    sc2_score.cc
+    sc2_server.cc
+    sc2_unit_filters.cc
+    sc2_unit.cc
+    "${PROJECT_SOURCE_DIR}/src/sc2api/typeids/sc2_${SC2_VERSION}_typeenums.cpp")
+
+add_library(sc2api STATIC ${sc2api_sources})
+
+target_include_directories(sc2api PUBLIC "${PROJECT_SOURCE_DIR}/include")
```

Comment:
This change cause small degradation: now we include all version headers while we need only two of them.
Not critical.